pwa/src/components/welcomeModal/welcomeModal.tsx

```diff
@@ -0,0 +1,57 @@
+
+import * as React from "react";
+import { navigate } from "gatsby";
+import APIService from "../../apiService/apiService";
+import APIContext from "../../apiService/apiContext";
+import { Modal } from "@conductionnl/nl-design-system";
+
+const WelcomeModal: React.FC = () => {
+  const API: APIService = React.useContext(APIContext);
+  const buttonRef = React.useRef(null);
+
+  React.useEffect(() => {
+    // Check route
+    if (window.location.href.split("/").at(-1) !== 'applications' || (window.location.href.split("/").at(-1) !== 'new' && window.location.href.split("/").at(-2) !== 'applications')) {
+      renderOptionalWelcomeModal()
+    }
+  }, [API, buttonRef])
+
+  const renderOptionalWelcomeModal = (): void => {
+    API.Application.getAll()
+      .then((res) => { buttonRef.current !== null && (res.data.length === 0 && buttonRef.current.click()); })
+      .catch((err) => { throw new Error('GET applications error: ' + err) })
+  }
+
+  const welcomeModalBody = (): JSX.Element => {
+    return (
+      <>
+        <p>Welcome to the gateway user-interface.</p>
+        <p>It seems you haven't created a application yet, would you like to
+          <a
+            className="utrecht--link"
+            data-bs-dismiss="modal"
+            aria-label="Close"
+            onClick={() => { navigate('/applications/new') }}>
+            {` create an application `}
+          </a>
+          first or you can close this modal if you want to find things out yourself. </p>
+      </>
```

Comment:
Generally speaking, a call to action is always written in an _active_ manner. This message is quite passive.

Current: "would you like to create an application ... "

Target (e.g.): "Click here to create your first application"

Also, the "or you can close this modal if you want to find things out yourself." might not be the best addition because the modal will pop-up on every screen. Maybe just be more direct and say that a user **has to** create an application to get started.